.github/workflows/scheduled.yml

```diff
@@ -12,19 +12,11 @@ jobs:
         fetch-depth: 0
     - uses: ruby/setup-ruby@v1
       with:
-        ruby-version: 3.0
```

Comment:
We don't need to pass in the `ruby-version` value, because the next thing the action will try is to check the `.ruby-version` file, and since we have that created, it'll just use that Ruby version automagically.